README.md

```diff
@@ -192,126 +174,22 @@ trainer = Trainer(logger=wandb_logger)
 - **[Try in a colab →](http://wandb.me/lightning)**
 - [Docs](https://docs.wandb.ai/guides/integrations/lightning)
 
+</details>
 
-## 🤗 HuggingFace
-Just run a script using HuggingFace's Trainer passing `--report_to wandb` to it
-in an environment where `wandb` is installed, and we'll automatically log losses,
-evaluation metrics, model topology, and gradients:
+&nbsp;
```

Comment:
Mention + add a link to a doc where all our integrations are mentioned? We have a bunch...

Reply:
Keep HuggingFace?